doorman/application.py

```diff
@@ -14,12 +15,13 @@
 from doorman.tasks import celery
 from doorman.utils import get_node_health, pretty_field, pretty_operator
 
-
 def create_app(config=ProdConfig):
     app = Flask(__name__)
     app.config.from_object(config)
     app.config.from_envvar('DOORMAN_SETTINGS', silent=True)
-
+    # if running on heroku, redirect http to https
+    if 'DYNO' in os.environ:
+        sslify = SSLify(app)
```

Comment:
This cannot be done until SSLify fix their initialization to work without app context, see PR https://github.com/kennethreitz/flask-sslify/pull/45